openapi/components/schemas/Timelines/CreditMemoTimeline.yaml

```
@@ -15,20 +15,20 @@ properties:
       - credit-memo-partially-applied
       - credit-memo-voided
   triggeredBy:
-    description: Shows who or what triggered the Timeline event.
+    description: Specifies who, or what, triggered the timeline event.
     type: string
     readOnly: true
     enum:
       - rebilly
       - app
       - direct-api
   message:
-    description: The message that describes the message details.
+    description: Contents of the timeline message.
```

```suggestion
    description: Content of the timeline message.
```
Not sure, just a suggestion